programs/invariant/invariant-types/src/math.rs

```diff
@@ -162,7 +162,11 @@ pub fn compute_swap_step(
         // take fee in input_amount
         // U256(2^64) * U256(1e12) - no overflow in intermediate operations
         // no overflow in token_amount result
-        let amount_after_fee = amount.big_mul(FixedPoint::from_integer(1u8) - fee);
+        let amount_after_fee = amount.big_mul(
+            FixedPoint::from_integer(1u8)
+                .checked_sub(fee)
+                .map_err(|_| err!("sub overflow"))?,
```

Comment:
A minor difference, but it is more correct to write "sub underflow" instead of "sub overflow"